strats.md

```diff
@@ -1075,6 +1084,55 @@ There are additional requirements depending on the exit condition:
 }
 ```
 
+### Come In With Blue Spring Ball Bounce
+
+A `comeInWithBlueSpringBallBounce` entrance condition indicates that Samus must enter the room by spring ball bouncing in from the previous room, while having blue speed. This is applicable to horizontal transitions. It has the following properties:
+
+- _adjacentMinTiles_: This is the minimum effective runway length in case an adjacent runway (connected to the door) is used to gain speed, jump, and bounce. This includes the entire length of runway from where the run begins, up to the door transition.
```

Comment:
Yeah, I was thinking about this, and in theory it would be cleaner that way. It's just that coming up with a formula for how to compute those unusable tiles seems non-trivial. I think we could have some kind of "short-hop mockball proficiency" that factors into it. So I guess there's a trade-off between the amount of extra detail needed in the applications/entrances, vs. how much hidden complexity is stuffed into behind-the-scenes math for how the cross-room conditions get resolved. I think we could always make a change later if we figure out that math and decide we want it to be automated like that.